api/authorization.yml

```diff
@@ -80,6 +88,7 @@ components:
 
   schemas:
     Pagination:
+      description: These parameters are required for lakeFS to display paginated results on the page.
```

Comment:
The pagination mechanism is used to retrieve a list of items.

Without `lakeFS` and without `display` it is very specific use-case.